bakerydemo/templates/base.html

```diff
@@ -18,6 +18,9 @@
         <meta name="description" content="{% if page.search_description %}{{ page.search_description }}{% endif %}">
         <meta name="viewport" content="width=device-width, initial-scale=1">
 
+        {# Prevent the demo site from being indexed #}
+        <meta name="robots" content="noindex" />
```

Comment:
Same here, this template is likely to be copy and pasted around so I'm not sure this belongs here.